README.md

```diff
@@ -36,3 +36,21 @@ To install, run the following with [npm](https://www.npmjs.com):
 ```
 npm install jira-precommit-hook
 ```
+
+**SYMLINK DIRECTORY NOTICE**
+
+If the hooks directory in your .git folder is symlinked, this hook will fail to install. To avoid this, keep the hooks directory inside of the project's .git directory.
```

Comment:
Sorry just noticed one more thing

> To avoid this, keep the hooks directory inside of the project's .git directory

Can you make this more clear? Since a symlink is technically in the .git directory.